source/main/modules/window-manager/index.ts

```diff
@@ -219,6 +224,53 @@ export default class WindowManager {
       return
     }
 
+    const platformIcons: {[prop: string] : string} = {
+      'darwin': '/png/16x16.png',
+      'win32': '/icon.ico'
+    }
+
+    this._mainWindow.on('show', async () => {
+      if (this._tray == null) {
+        if (process.platform === 'linux') {
+          let size = '16'
+          const sizeList = ['16', '24', '32', '48', '64', '96', '128', '256', '512']
+          const display = screen.getPrimaryDisplay()
+          if (process.env.XDG_CURRENT_DESKTOP === 'GNOME') {
+            size = display.workArea.y + ''
+          } else if (process.env.XDG_CURRENT_DESKTOP === 'KDE'){
+            size = display.size.height - display.workArea.height + ''
+          }
+          if (sizeList.includes(size)) {
+            this._tray = new Tray(path.join(__dirname, `assets/icons/png/${size}x${size}.png`))
+          } else {
+            this._tray = new Tray(path.join(__dirname, `assets/icons/png/16x16.png`))
+          }
+        } else {
+          this._tray = new Tray(path.join(__dirname, 'assets/icons', platformIcons[process.platform] || '/png/16x16.png'))
+        }
+        
+        const contextMenu = Menu.buildFromTemplate([
+          {
+            label: 'Show Zettlr',
+            click: () => {
+              this.showAnyWindow()
+            },
+            type: 'normal'
+          },
+          { label: '', type: 'separator' },
+          {
+            label: 'Quit',
+            click: () => {
+              app.quit()
+            },
+            type: 'normal'
+          }
+        ])
+        this._tray.setToolTip('This is Zettlr tray. \n click show Zettlr button to display app \n click quit to quit app')
```

Comment:
I suggest: `This is the Zettlr tray.` Upper case Z in Zettlr.
I suggest: `Select Show Zettlr to show the Zettlr app.` Use Select because the user may not click. The user may be using a touch device. Upper case S in Show to match the label `Show Zettlr`. Use the same verb 'show'.
I suggest: `Select Quit to quit the Zettlr app.`

I suggest:
```typescript
this._tray.setToolTip('This is the Zettlr tray. \n Select Show Zettlr to show the Zettlr app. \n Select Quit to quit the Zettlr app.')
```